CMakeLists.txt

```diff
@@ -495,6 +492,12 @@ if(WITH_RADOSGW)
   endif()  # OPENSSL_FOUND
 endif (WITH_RADOSGW)
 
+#option for ceph-mds
+option(WITH_MDS "build ceph-mds daemon" ON)
+
+# building libcephfs needed for mds 
```

Comment:
i don't think libcephfs is a dependency of mds. the former is the client side library of cephfs, while the latter is the server implementation specific to cephfs.

Reply:
@tchaikov 
So libcephfs is only used in the clients?
I'll augement the comment as such